### PR TITLE
Reorganize folder structure

### LINKED ISSUE #1214: Better folder structure
**Software** eggs are in the **software** folder.
**Storage** eggs are in the **storage** folder.
**Database** eggs are in the **database** folder.
And **Bots** are in the **bots** folder.

But **Game Eggs** are completely random distributet over the whole root of the repository.

It confuses me and also doesn't look that good. All eggs should be in their corresponding category. Like **Game Eggs** in a _games_ directory.

I would change it and make a contribution so it's more clear and better understandable.

### PR DESCRIPTION
**Pull request by Software-Noob**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

Resolves #1214

Moves all game eggs inside game_eggs folder. This makes navigating the repo way easier and displays the main readme right away. Updated all eggs that pull files from the repo, together with all the readme files. Deleted duplicate tower_unite egg.

### CODE REVIEW

**Comment by DatMayo** [comment]
Just an addition: You named the folder `game_eggs`, other things are called `database` or `voice_server`. To keep things structured, you could rename them to i.E. `database_eggs`, `voice_eggs`, ...

**Comment by Software-Noob** [comment]
> Just an addition: You named the folder `game_eggs`, other things are called `database` or `voice_server`. To keep things structured, you could rename them to i.E. `database_eggs`, `voice_eggs`, ...

The only reason I named it `game_eggs`  is to make it stand out from the rest as game eggs are the priority, similar to the stock_eggs

**Comment by DatMayo** [comment]
> > Just an addition: You named the folder `game_eggs`, other things are called `database` or `voice_server`. To keep things structured, you could rename them to i.E. `database_eggs`, `voice_eggs`, ...
> 
> The only reason I named it `game_eggs` is to make it stand out from the rest as game eggs are the priority, similar to the stock_eggs

Okay, I see. But I think it would be nice to have it.

File: game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together (1).json [hunk 1]
```diff
@@ -1,13 +1,18 @@
 {
```
Comment (parkervcp) [inline game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together (1).json:1]: why is there a (1) in the json name?

**Review by parkervcp — CHANGES_REQUESTED** [review]

File: game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together (1).json (hunk 1 above)
Comment (Software-Noob) [reply game_eggs/steamcmd_servers/dont_starve/egg-don-t-starve-together (1).json]: Had to fix one missed config pull in the install script and didn't notice a duplicate file.

**Comment by parkervcp** [comment]
@Software-Noob We will need to PR changes to the panel repo because some of those eggs make calls for the server/properties.

We should leave some of those around like the minecraft and a few others.

**Comment by Software-Noob** [comment]
> @Software-Noob We will need to PR changes to the panel repo because some of those eggs make calls for the server/properties.
> 
> We should leave some of those around like the minecraft and a few others.

The only stock egg that fetches any files from the stock eggs is paper for the server.properties file - we could leave the Minecraft folder that contains the file for backward compatibility

**Comment by parkervcp** [comment]
> > @Software-Noob We will need to PR changes to the panel repo because some of those eggs make calls for the server/properties.
> > We should leave some of those around like the minecraft and a few others.
> 
> The only stock egg that fetches any files from the stock eggs is paper for the server.properties file - we could leave the Minecraft folder that contains the file for backward compatibility.
> 
> Unless you'd like to leave all the files that other eggs are fetching too. It's around 30 eggs I believe, the majority being Minecraft.

Just the server.properties file is fine for now.

**Comment by parkervcp** [comment]
@Software-Noob in the games dir can you move beammp under beamng next to kissmp?

**Review by parkervcp — APPROVED** [review]
LGTM

**Pull request merged**